plugin/emacs/test/quicklintjs-test.el

```diff
@@ -9,6 +9,12 @@
                           (expand-file-name default-directory)
                           ".melpa-cache/"))
 
+(defun quicklintjs-install-deps (deps)
+  (mapcar (lambda (pkg) (unless (package-installed-p pkg)
+                          (if (> emacs-major-version 24)
+                              (package-install pkg t)
+                            (package-install pkg)))) deps))
```

Comment:
Nit: Indentation looks off to me. Shouldn't there be two more spaces before this line to align it with line 15?

Reply:
I've been indenting with a script that runs emacs commad "indent-region" in the file, it does indent as you're looking, and lisp code inside emacs /lisp directory is also indented like this:
```lisp
 (if (> 0 0) ; cond
     (foo) ; then
   (bar)   ; else
```
so i think it's correct according to Emacs standards